types/ember__component/test/helper.ts

```diff
@@ -65,6 +65,26 @@ class BadReturnForm extends Helper<DemoSig> {
     }
 }
 
+class NoSignature extends Helper {
+    compute([i18nizer]: [i18nizer: (s: string) => string], { name, age }: { name: string; age: number }): string {
```

Comment:
With this change, you end up with `NoSignature` extending `Helper<DefaultSignature>` despite the actual `compute` method having a narrower type, don't you?

That would allow this helper to be invoked with any arbitrary arguments in Glint, which probably isn't what the author would expect

Reply:
Yes. Providing a signature would be necessary for Glint to work properly. This would just be to not break type-checking for those not using Glint.